cypress/locators/commonLocators.js

```diff
@@ -6,6 +6,11 @@ const LABEL_HELP_INPUT = '#labelHelp'
 const TITLE_INPUT = '#title'
```

Comment:
Prettier format (semicolons, spaces)

Reply:
for common files semicolons and other will be fixed in future to avoid conflicts now

Reply:
sure :)